server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -96,4 +86,21 @@ class AddressIndexRepository @Inject()(conf : AddressIndexConfigModule, elasticC
       )
     }
   }.map(response => PostcodeAddressFileAddresses(response.as[PostcodeAddressFileAddress], response.maxScore))
+
```

Comment:
Repeat code is always bad.

expose buildingNumberFieldname and postcodeFieldName as arguments.